examples/obs-agent-iks/variables.tf

```diff
@@ -39,3 +39,9 @@ variable "datacenter" {
   description = "If creating a classic cluster, the data center where the cluster is created"
   default     = "syd01"
 }
+
+variable "prometheus_config" {
```

Comment:
Its probably better to hard code a sample value in the example itself instead of exposing a variable. That way people can see a sample value that can be passed. We don't usually expose variables like this in examples as we don't want the example to become flexible solution thats people will use. they are just for reference